### ISSUE TITLE
Should b_stty() be enabled or removed?

### ISSUE THREAD

**Issue by krader1961** [att/ast#744]
Issue #129 discussed whether or not we want to, or should, support a lot of builtin versions of commonly used external commands. The final decision was to limit the builtins to those found in the ksh93u+ release and remove those added later. While working on #738 I noticed that `b_stty()` is the only builtin added after ksh93u+ that is still being built but is not present in the `shtab_builtins[]` table. Should we include it or remove the code?

**Comment by krader1961** [comment]
I'm going to delete the stty implementation. I hate to throw away code that (probably) works but `stty` is used so seldom these days that it doesn't make sense to retain it as a builtin. Even in the past when it was fairly common for UNIX users to run the command it was never run frequently enough to justify being a builtin. Other builtins added after the ksh93u+ release are much easier to justify. For example, `chown`, `chmod`, etc. are frequently used in shell scripts. And commands like `realpath` may not even be available on a given system. Those are much easier to justify as builtins than `stty`.

**Comment by dannyweldon** [comment]
Yes, we don't bother running it in `.profile` now and the only use now I can think of is for disabling echo using `stty -echo`, for inputting passwords.  It bugs me though, that an external command is needed to control something like that, as I wish it could be done with something like `set -o noecho` instead.  I would be happy if a feature like that would be added to replace `stty`, and I imagine that reading the current value of the option would require querying the current state of the terminal driver to take into account any side effects of running external commands.

It's just my opinion.  Feel free to disagree, comment, etc.  :-)
